sunkit_spex/extern/rhessi.py

```diff
@@ -1,6 +1,8 @@
 import copy
 import warnings
 
+import sunpy.time
```

Comment:
register `utime` with astropy Time

Reply:
what is utime?

Reply:
UT seconds from 1979-01-01 00:00:00 UTC, ignoring leap seconds (https://docs.sunpy.org/en/stable/generated/api/sunpy.time.TimeUTime.html)